draft-kazuho-httpbis-priority.md

```diff
@@ -300,6 +302,84 @@ receipt, a client that supports header-based prioritization MUST close the
 connection with a protocol error. Non-supporting clients will ignore this
 extension element (see {{!RFC7540}}, Section 5.5).
 
+# Reprioritization
+
+Once a client sends a request with a priority, circumstances might change and
+mean that it is beneficial to change the priority of the response. As an
+example, a web browser might issue a prefetch request for an HTML document with
+the urgency parameter of the Priority request header field set to `background`.
+Then, when the user navigates to the HTML while prefetch is in action, it would
+send a reprioritization frame with the priority field value set to `urgency=0`.
+
+However, a client cannot reprioritize a response by using the Priority header
+field.  This is because an HTTP header field can only be sent as part of an HTTP
+message. Therefore, to support reprioritization, it is necessary to define a
+HTTP-version-dependent mechanism for transmitting the priority parameters.
+
+This document specifies a new frame type for HTTP/2 ({{?RFC7540}}) and HTTP/3
+({{?I-D.draft-ietf-quic-http-23}}) that is specialized for reprioritization. It
```

Comment:
Maybe we need to refer to them in a normative fashion (`!`) as we are defining extensions of the two?

Reply:
fixed, and tidied up all references to QUIC and HTTP/2 docs